test/cluster.jl

```diff
@@ -27,6 +27,11 @@ end
 const JOB_TEMPLATE = Mustache.load(joinpath(@__DIR__, "job.template.yaml"))
 const TEST_IMAGE = get(ENV, "K8S_CLUSTER_MANAGERS_TEST_IMAGE", "k8s-cluster-managers:$TAG")
 
+# Add a labels for easy cleanup of test resources. Any changes to these common labels also
+# needs to occur in the YAML files included in the test directory
+# e.g. `kubectl delete pod,job,sa,role,rolebinding -l k8s-cluster-managers-tests=true`
+const COMMON_LABELS = ("k8s-cluster-managers-tests" => "true",)
```

Comment:
```suggestion
const COMMON_LABELS = ("tests-k8s-cluster-managers" => "true",)
```
...any reason to put the "tests" first, so that if there are other sets of tests running at the same time, labels can be parsed/matched by searching for "tests-*"?

Reply:
K8s [label selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/) don't allow globing so this change won't help there. I'm not opposed making this change but usually how you accomplish this with K8s is to use multiple labels or in this case we could do:
```suggestion
const COMMON_LABELS = ("julia-test" => "K8sClusterManagers.jl",)
```

Reply:
oh, interesting. i haven't used labels enough to have a real opinion on this, but your proposed alteration does seem like it would make additional testing nicer (e.g., if we want to do BeaconK8sUtilities tests or anything).